cmd/thanos/query.go

```diff
@@ -141,6 +144,17 @@ func registerQuery(app *extkingpin.App) {
 			return errors.Errorf("Address %s is duplicated for --rule flag.", dup)
 		}
 
+		// Check if the YAML configuration of request.logging is correct. Exit early if error.
+		HTTPlogOpts, err := logging.DecideHTTPFlag(*reqLogDecision, reqLogConfig)
+		if err != nil {
+			return errors.Errorf("config for request logging not recognized %v", err)
```

Comment:
please don't use `errors.Errorf` when we wrap error. Use  `errors.Wrap` or  `errors.Wrapf` (same below)

Reply:
Also how we know reason? We just create decider... I think it should be decider that should give us this message from inside, no?

Reply:
Can you elaborate on thhis?

> Also how we know reason? We just create decider... I think it should be decider that should give us this message from inside, no?

Reply:
Done